src/Features/CSharp/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.cs

```diff
@@ -505,16 +505,16 @@ private static SyntaxNode GetTopMostParentingExpression(ExpressionSyntax express
             Debug.Assert(originalIdentifierNodes.Count() == inlinedExprNodes.Count());
 
             var originalNodesEnum = originalIdentifierNodes.GetEnumerator();
-            var inlinedNodesEnum = inlinedExprNodes.GetEnumerator();
+            var inlinedNodesOrTokensEnum = inlinedExprNodes.GetEnumerator();
```

Comment:
Where the enumerator is disposed? It might be required if it comes (or could be coming in future) from an iterator method with finally blocks.